spec/handler/zanox_spec.rb

```diff
@@ -96,7 +105,6 @@ def env
 
       it 'will display the correct tracking events' do
         expect(subject).to include 'window._zx.push({"id": "12345678D2345"});'
-        expect(subject).to include "var zx_category = 'Sewing';\nvar zx_identifier = '234';\nvar zx_amount = '5.90';"
```

Comment:
to_json wraps the values with double quotes instead of single quotes. so this should stay, but the expected value should use double quotes